mmv1/third_party/terraform/services/servicenetworking/resource_service_networking_connection_test.go

```diff
@@ -56,7 +56,7 @@ func TestAccServiceNetworkingConnection_abandon(t *testing.T) {
 			{
 				ResourceName:      "google_service_networking_connection.foobar",
 				ImportState:       true,
-				ImportStateVerify: true,
```

Comment:
lets use https://github.com/GoogleCloudPlatform/magic-modules/blob/a5036e705278daac790cab6f7127038cf924155a/mmv1/third_party/terraform/services/bigtable/resource_bigtable_app_profile_test.go#L30

to ignore the offending field instead of disabling this.